modules/special-rules/pages/weapon-mount-examples.adoc

```diff
@@ -0,0 +1,66 @@
+= Weapon mount examples
+
+.War engine weapon mounts and weapon arcs
+====
+Refer to the relevant data sheets and you will see that:
+
+* The Imperial Warlord Titan has four weapon mounts: Left Arm; Right Arm; Left Carapace; Right Carapace.
+* The Imperial Baneblade super-heavy tank has one weapon mount: Turret Guns.
+* The Imperial Shadowsword super-heavy has two weapon mounts: {death-ray}; Sponson Turrets.
+// TODO: It's not a 'Death Ray mount' — it's an 'X' ('Hull' or 'Volcano Cannon'?) mount that holds a Death Ray super-heavy weapon (or provides a Death Ray shot, depending on how we define mounts to cover the various cases). We need to decide what we're calling the mounts and then apply the appropriate name here. UPDATE: If my mount+arrays idea works, we can overhaul these examples to match.
+
+You will also see that each weapon mount has an icon to indicate its weapon arc.
+====
+
+.Firepower weapon mounts: Warhound Titan
+====
+The Warhound Titan has two weapon mounts, which use the same weapon arc.
+Each weapon mount holds a Heavy Weapon Battery that has a Firepower value of 6.
+
+[NOTE]
+=====
+In all cases, you may only allocate hits to units that are actually inside the weapon arc.
+
+{blank}[TODO: This is the implication I think, but I worry that it can be abused by the type of shameless player that deliberately makes sure that an arc is 'just so', to pick off a juicy unit that just happens to be poking out a bit farther than other closer units in the target detachment.]
+=====
+
+.Targetting with separate weapon mounts
+* You may shoot both weapon mounts at the same target detachment, to direct a total of 12 Firepower at it.
+You may shoot each weapon mount at a different target detachment, to direct 6 Firepower at each of them.
+
+.Targetting with an individual weapon mount
+* You may only shoot each weapon mount at one detachment -- that is, you may not split fire with an individual weapon mount.
+
+.Weapon mounts and suppression by Blast markers
+* For each Blast marker on the Warhound Titan, reduce its Firepower by one.
+You may choose which weapon mount loses the Firepower.
+{blank}[TODO: Actually, I think I have this wrong -- isn't it -1 Firepower per target detachment?]
```

Comment:
```suggestion
.Weapon mounts and suppression by Blast markers
* For each Blast marker on the Warhound Titan, reduce its Firepower by one.
You may choose which weapon mount loses the Firepower.
{blank}[TODO: Actually, I think I have this wrong -- isn't it -1 Firepower per target detachment?]
```
It's mostly right.

Each BM reduces the Warhound's total Firepower, no matter how it splits that Firepower. So 1 BM and splitting fire would result in 5 FP against one target and 6 FP against the other target.

Where your text is _maybe_ a bit ambiguous is that it can be read as having to apply all BMs to the same mount, which isn't the case - the player can choose where each individual BM gets applied. 

Example options with 4 BMs and splitting fire:
- 6 FP and 2 FP
- 5 FP and 3 FP
- 4 FP and 4 FP

Reply:
>Each BM reduces the Warhound's total Firepower, no matter how it splits that Firepower. So 1 BM and splitting fire would result in 5 FP against one target and 6 FP against the other target.

What was nagging me is that I could've sworn I saw something somewhere to the effect that if you split fire then you get the disincentive of suppression on both parts. But if not then all is well, the way I'd written it already works I think?

>Where your text is maybe a bit ambiguous is that it can be read as having to apply all BMs to the same mount, which isn't the case - the player can choose where each individual BM gets applied.

I thought I'd covered that, but I see what you mean -- the way I phrased it could be read to imply that you can choose the mount but then all the of the suppression goes on that one mount. Well spotted.